homeassistant/components/homeassistant/strings.json

```diff
@@ -121,6 +121,10 @@
     "save_persistent_states": {
       "name": "Save persistent states",
       "description": "Saves the persistent states immediately. Maintains the normal periodic saving interval."
+    },
+    "reload_all": {
+      "name": "Reload all",
+      "description": "Reloads all of the YAML configurations."
```

Comment:
This is misleading. The [documentation](https://www.home-assistant.io/integrations/homeassistant/#service-homeassistantreload_all) says:
> Reload all YAML configuration that can be reloaded without restarting Home Assistant.

I think that's a much better summary of what the service does

Reply:
Thank you, I fixed it.